pkg/model/elbv2/load_balancer.go

```diff
@@ -78,6 +78,23 @@ const (
 	LoadBalancerSchemeInternetFacing                    = "internet-facing"
 )
 
+const (
+	DeletionProtectionEnabledKey = "deletion_protection.enabled"
+	AccessLogsS3EnabledKey       = "access_logs.s3.enabled"
+	AccessLogsS3BucketKey        = "access_logs.s3.bucket"
+	AccessLogsS3PrefixKey        = "access_logs.s3.prefix"
+	IdleTimeoutTimeoutSecondsKey = "idle_timeout.timeout_seconds"
+	RoutingHTTP2EnabledKey       = "routing.http2.enabled"
+	CrossZoneLoadBalancing       = "load_balancing.cross_zone.enabled"
+
+	DefaultDeletionProtectionEnabled = false
```

Comment:
we shouldn't have these default value in the model package.
it should be inside relevant model builder